spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/Engine.scala

```diff
@@ -327,6 +350,12 @@ object Engine {
       _model = new ThreadPool(modelPoolSize)
       _model.setMKLThread(MKL.getMklNumThreads)
     }
+
+    if (engineType == MklDnn) {
```

Comment:
The code is similar to what `setMKLThread` does for each thread; maybe put it into a method and then invoke the method

Reply:
Yes. I'll move them to a new methods in `ThreadPool` object.